src/main/java/com/minecolonies/sounds/FishermanSounds.java

```diff
@@ -0,0 +1,112 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.entity.EntityCitizen;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+/**
+ * Class containing the fisherman sounds.
+ */
+public final class FishermanSounds
+{
+    /**
+     * Random generator.
+     */
+    private static Random rand = new Random();
+
+    /**
+     * Number of different sounds in this class.
+     */
+    private static final int NUMBER_OF_SOUNDS = 2;
+
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 25;
+
+    /**
+     * Chance to play a basic sound.
+     */
+    private static final int BASIC_SOUND_CHANCE = 100;
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private FishermanSounds()
+    {
+        /*
+         * Intentionally left empty.
+         */
+    }
+
+    /**
+     * Containing the female fisherman sounds.
+     */
+    public static final class Female
+    {
+        public static final String generalPhrases = Constants.MOD_ID + ":mob.fisherman.female.generalPhrases";
+        public static final String noises = Constants.MOD_ID + ":mob.fisherman.female.noise";
+        public static final String iGotOne = Constants.MOD_ID + ":mob.fisherman.female.iGotOne";
+        public static final String needFishingRod = Constants.MOD_ID + ":mob.fisherman.female.needFishingRod";
+        public static final String offToBed = Constants.MOD_ID + ":mob.fisherman.female.offToBed";
+        public static final String badWeather = Constants.MOD_ID + ":mob.fisherman.female.badWeather";
+
+        /**
+         * Private constructor to hide the implicit public one.
+         */
+        private Female()
+        {
+            /*
+              Intentionally left empty.
+             */
+        }
+    }
+
+    /**
+     * Containing the male fisherman sounds.
+     */
+    public static final class Male
+    {
+        public static final String generalPhrases = Constants.MOD_ID + ":mob.fisherman.male.generalPhrases";
+        public static final String noises = Constants.MOD_ID + ":mob.fisherman.male.noise";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00115)